packages/prisma-client/migrations-cli/commands.ts

```diff
@@ -256,6 +276,29 @@ If you're sure no other process is running, please delete the lockfile:
   }
 };
 
+export const migrate = async () => {
+  const status = await getStatus();
+
+  ensureNoFailed(status);
+
+  if (status.pending.length === 0) {
+    logger.info("No pending migrations\n");
+    process.exit(0);
+  }
+
+  if (args.dev) {
+    logger.info("You're about to apply the following migration(s):");
```

Comment:
Btw I think this debug package would be even better than custom logger https://www.npmjs.com/package/debug

Reply:
The only reason we need a custom logger is to be able to print CLI messages while still using `no-console` rule. We don't need any other features.

Reply:
debug is not just console.log replacement, its about ability to control what will be logged using DEBUG env variable that you can pass before the call, e.g. I could do something like this DEBUG=migrations yarn migrations and it would log all the info for debugging that would be otherwise silent

or the other way around, if your output is not silent, I can silence it for CLI for example by DEBUG=*,-not_this

but again, just an idea

Reply:
btw this debug package is used by almost every popular package, you can try just putting DEBUG=prisma:engine

Reply:
I think `debug` package is specifically for the debug output, basically a replacement for `console.debug()` but not for `console.info()`.

Overall I think it's a bit wasteful to focus on these little things right now. I'd just keep as is and forget about it unless there's a strong reason to change something.